bran/bran.py

```diff
@@ -435,7 +181,7 @@ def main():
     print("\ninstance", instance_id[0], "initialized")
     print("installing necessary software..")
     if purpose_answer["purpose"] == "RavenML Training":
-        time.sleep(100)
+        countdown(235) #for gpu the dependencies take a while to download
```

Comment:
is 235 just based on the time you usually observe it takes to download these dependencies?

Reply:
I've never actually timed it out. It used to be 180s and occasionally dependencies would still be installing when I sshd. I'll go back and time it out.